README.md

```diff
@@ -2,48 +2,60 @@
 
 CodePath WEB103 Final Project
 
-Designed and developed by: [your names here]
+Designed and developed by: Miho Shimizu, Jiarui Liang
 
 🔗 Link to deployed app:
 
 ## About
 
 ### Description and Purpose
 
-[text goes here]
+[App Name] helps people manage and monitor food items they have in thier refridgerator. This app allows users to keep track of what food items they have, their quantities, and expiration dates. Users can share the information among their housemates. This food tracking system facilitates to use their food resources efficiently, reduce food waste, and plan their meals based on the ingredients they have available. It's a handy tool for households, community organizations, and anyone interested in optimizing their food inventory.
 
 ### Inspiration
 
-[text goes here]
+You're all set to cook that favorite recipe, only to find that a crucial ingredient you were sure you had, has magically disappeared - thanks to a helpful housemate! (Sometimes it's your yesterday self.) We've all been there, right? That relatable frustration sparked the idea for our app. We wanted to create a handy solution for managing your pantry and fridge to avoid those letdowns. Whether you're the master chef of the house or just someone tired of unwanted surprises, [App Name] is your answer to lessen "Who used the last egg?" moments and foster a more harmonious kitchen.
 
 ## Tech Stack
 
-Frontend:
+Frontend: React
 
-Backend:
+Backend: Node.js, Express.js, PostgreSQL (Railway)
 
 ## Features
 
-### [Name of Feature 1]
+### Fridges
 
-[short description goes here]
+Fridge is the unit of food inventory. Users can create and delete fridges. Within each fridge, users can manage food items by adding, updating, and removing. Additionally, users can manage members to share the info with by adding and removing. 
 
 [gif goes here]
 
-### [Name of Feature 2]
+### Food items
 
-[short description goes here]
+Food items have attributes such as count, expiration date, and category. When the food expires, the item will be highlighted. Users can also mark missing food, so the members know that they need to buy them.
 
 [gif goes here]
 
-### [Name of Feature 3]
+### Members
 
-[short description goes here]
+Fridge members share the fridge info. Users can be a member of multiple fridges. 
 
 [gif goes here]
 
 ### [ADDITIONAL FEATURES GO HERE - ADD ALL FEATURES HERE IN THE FORMAT ABOVE; you will check these off and add gifs as you complete them]
 
+### Shopping Lists
+
+Users have access to a shopping list that aggregates all missing food items to be bought.
+
+[gif goes here]
+
+### Dash Board
+
+Users can create or access fridges from this page.
```

Comment:
So the dashboard is a panal for creating and accessing fridges, right?

Reply:
Yes